website/docs/r/auth_backend.html.md

```diff
@@ -28,7 +28,7 @@ The following arguments are supported:
 
 * `type` - (Required) The name of the auth method type
 
-* `path` - (Optional) The path to mount the auth method — this defaults to the name of the type
+* `path` - (Optional) The path to mount the auth method — this defaults to the name of the type.
```

Comment:
nit: while we're here could we also update the docs for `type` and `description` to also have periods at the end for consistency? No big deal though

Reply:
Sounds good.

Reply:
Fixed with https://github.com/hashicorp/terraform-provider-vault/pull/1471/commits/c31cb428d18b64ac5519563bb1ec37f9aee6e2fd